experiment/test_config.yaml

```diff
@@ -1057,7 +1057,7 @@ jobs:
     args:
     - --env=KUBE_PROXY_DAEMONSET=true
     - --env=ENABLE_POD_PRIORITY=true
-    - --env=KUBE_FEATURE_GATES=AllAlpha=true
+    - --env=KUBE_FEATURE_GATES=AllAlpha=true,CSIMigration=false
```

Comment:
I think you will need to regenerate the generated jobs, ref https://github.com/kubernetes/test-infra/blob/88bc9f98f299e0fddfe1812d127d451d7fa8b0e2/experiment/test_config.yaml#L15-L18

Reply:
done, thanks for the tip